public class tryCatchException {

    public void method(){
        try{

        }
        catch(ArithmeticException e){
            System.exit(1);

        }

        finally{

        }
    }
}
